_posts/05-04-01-Design-Patterns.md

```diff
@@ -1,18 +1,18 @@
 ---
 isChild: true
-title: Dizajn paterni (šabloni)
+title: Dizajn šabloni (Design patterns)
 anchor: design_patterns
 ---
 
-## Dizajn paterni (šabloni) {#design_patterns_title}
+## Dizajn šabloni (Design patterns) {#design_patterns_title}
 
-Kada pravite aplikaciju biće vam od pomoći ako koristite uobičajene šablone (paterne) u svom kodu i uobičajene
-paterne za kompletnu strukturu projekta. Korišćenje uobičajenih paterna će vam pomoći jer pojednostavljuje upravljanje
-vašim kodom i omogućava drugim programerima da brže sagledaju kako se sve uklapa.
+Pri razvoju vaše aplikacije korisno je upotrebljavati neke uobičajene šablone (pattern-e) u kôdu,
+kao i šablone po pitanju same strukture projekta. Korišćenje i primena pattern-a je korisna jer pojednostavljuje
+upravljanje vašim kôdom i omogućava drugim programerima da brže shvate kako je sve organizovano i uklopljeno.
 
-Ako koristite frejmvork onda će većina komplikovanijeg koda i struktura projekta biti bazirana na tom frejmvorku, tako
-da će dosta odluka oko paterna biti doneto umesto vas. Ali ostaje na vama da izaberete najbolje paterne koji će pratiti
-kod koji gradite na bazi tog frejmvorka. Ako, s druge strane, ne koristite frejmvork za izradu aplikacije onda morate
-izabrati paterne koji najbolje odgovaraju tipu i veličini aplikacije koju pravite.
+Ako koristite framework, onda će većina kôda višeg nivoa i struktura projekta biti bazirana na tom framework-u, tako
+da će dosta posla biti urađeno umesto vas. Ali na vama je i dalje da primenjujete patern-e u kôdu kojeg ćete pisati
```

Comment:
Da, namerno tu sam svesno izbacio neke stvari i nisam se baš toliko pridržavao originala, gledao sam nekako da probam svojim rečima da opišem to.

Ne znam, ja koliko sam shvatio, tu je pisac hteo da kaže da je sam framework ima neka svoja pravila i specifičnosti koje developer mora da isprati. Tipa struktura projekta, arhitektura, MVC koncepcija i slično.